src-js/rollbar.js

```diff
@@ -0,0 +1,25 @@
+import * as rollbar from 'rollbar-browser';
+
+export const init = (environment = 'production') => {
```

Comment:
Is this supposed to be a single equals operation here? 
Are you meaning to assign the environment as production? Or test it, as below?